packages/driver/src/cy/commands/cookies.ts

```diff
@@ -79,28 +79,16 @@ function cookieValidatesSecurePrefix (options) {
   return options.secure === false
 }
 
-interface InternalGetCookieOptions extends Partial<Cypress.Loggable & Cypress.Timeoutable> {
-  _log?: Log
-  cookie?: Cypress.Cookie
-}
-
-interface InternalGetCookiesOptions extends Partial<Cypress.Loggable & Cypress.Timeoutable> {
-  _log?: Log
-  cookies?: Cypress.Cookie[]
-}
-
-interface InternalSetCookieOptions extends Partial<Cypress.SetCookieOptions> {
-  _log?: Log
-  name: string
-  cookie?: Cypress.Cookie
-}
-
-type InternalClearCookieOptions = InternalGetCookieOptions
-
-interface InternalClearCookiesOptions extends Partial<Cypress.Loggable & Cypress.Timeoutable> {
-  _log?: Log
-  cookies?: Cypress.Cookie[]
-  domain?: any
+function validateDomainOption (domain: any, commandName: string, log: Log | undefined) {
+  if (domain !== undefined && domain !== null && !_.isString(domain)) {
```

Comment:
I am wondering, do we need to add stricter logic to ensure the passes string matches domain patterns? I assume this should not include the schema? or the subdir aspects of a url?